SQL/Insert_PlaceholderData_Activity.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO ACTIVITY (ActivityId, PetId, TimeStamp, Type) VALUES ('1', '1', '03/07/2018 10:50:50 PM', 'd');
+INSERT INTO ACTIVITY (ActivityId, PetId, TimeStamp, Type) VALUES ('2', '2', '03/08/2018 10:50:50 PM', 'd');
```

Comment:
Don't need to specify the activity type for all of these statements

Also, integers are not chars.